static/js/utils/tests/chart_csv_utils.test.ts

```diff
@@ -310,17 +312,46 @@ test("rankingPointsToCsv", () => {
           value: 3,
         },
       ],
+      dataHeadings: ["data"],
       expected: [
         "rank,place,data",
         "1,placeAName,1",
         "2,placeBId,2",
         "3,placeCName,3",
       ].join("\n"),
     },
+    {
+      name: "multi column",
+      rankingPoints: [
+        {
+          placeDcid: "placeAId",
+          placeName: "placeAName",
```

Comment:
Maybe add some space and `,` to the name, and we should expect the cell to be wrapped in double quotes?

Reply:
done